src/go/rpk/pkg/api/admin/admin_test.go

```diff
@@ -110,3 +110,11 @@ func TestListUsers(t *testing.T) {
 	require.NoError(t, err)
 	require.Exactly(t, []string{"Joss", "lola", "jeff", "tobias"}, users)
 }
+
+func TestProperytMetadata(t *testing.T) {
+	raw := []byte(`{"description": "Recover raft configuration on start for NTPs matching pattern", "nullable": false, "needs_restart": true, "visibility": "tunable", "type": "array", "items": {"type": "string"}}`)
+
+	var unmarshaled ConfigPropertyMetadata
+	err := json.Unmarshal(raw, &unmarshaled)
+	require.NoError(t, err)
```

Comment:
This was testing what happened if ConfigPropertyMetadata expected a string for Items rather than an object (which was a mistake I made when first writing it).

But yeah, without that context it's kind of a random thing to test, so I'll just remove this, it's covered in the ducktape tests anyway.